src/go/k8s/pkg/console/configmap.go

```diff
@@ -146,6 +146,8 @@ func (cm *ConfigMap) generateConsoleConfig(
 		return "", err
 	}
 
+	consoleConfig.Cloud = cm.genCloud()
```

Comment:
Can we put this [here](https://github.com/redpanda-data/redpanda/pull/6740/files#diff-622d2275ef6540ff881c27df273e8576383c794dbb32d7a868e7265950213735R130-R131)